src/plan_runner/plans/iiwa_plan_factory.cc

```diff
@@ -0,0 +1,55 @@
+#include "iiwa_plan_factory.h"
+#include "drake/common/trajectories/piecewise_polynomial.h"
+#include "drake/multibody/parsing/parser.h"
+#include "drake/common/find_resource.h"
+
+#include "joint_space_plan.h"
+
+using std::cout;
+using std::endl;
+using drake::trajectories::PiecewisePolynomial;
+using Eigen::MatrixXd;
+using Eigen::VectorXd;
+
+IiwaPlanFactory::IiwaPlanFactory() {
+  plant_ = std::make_unique<drake::multibody::MultibodyPlant<double>>(1e-3);
+  auto parser = drake::multibody::Parser(plant_.get());
+  std::string iiwa_path =
+      "drake/manipulation/models/iiwa_description/iiwa7/iiwa7_no_collision.sdf";
+  iiwa_path = drake::FindResourceOrThrow(iiwa_path);
+  parser.AddModelFromFile(iiwa_path);
+  plant_->WeldFrames(plant_->world_frame(),
+                     plant_->GetFrameByName("iiwa_link_0"));
+  plant_->Finalize();
+}
+
+
+std::unique_ptr<PlanBase>
+IiwaPlanFactory::MakePlan(const drake::lcmt_robot_plan &msg_plan) const {
+  // TODO: replace this with a better test and use an lcm type with an
```

Comment:
I agree :+1: we'll have to do this on our next round of PRs when Mark and I do new plans.